Elevel/Domain/Enums/Level.cs

```diff
@@ -8,10 +8,10 @@ namespace Elevel.Domain.Enums
 {
     public enum Level : byte
     {
-        Elementary,
-        PreIntermediate,
-        Intermediate,
-        Upperintermediate,
-        Advanced
+        Elementary = 0,
```

Comment:
It's better to use 0 for Not Applied (NA) or something like this, because it's usually default value

Reply:
Done.